[2023-03-28 22:53:44,393] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: country_dag.insert_task manual__2023-03-28T22:48:39.678113+00:00 [queued]>
[2023-03-28 22:53:44,403] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: country_dag.insert_task manual__2023-03-28T22:48:39.678113+00:00 [queued]>
[2023-03-28 22:53:44,403] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-28 22:53:44,403] {taskinstance.py:1357} INFO - Starting attempt 2 of 4
[2023-03-28 22:53:44,403] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-28 22:53:44,418] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): insert_task> on 2023-03-28 22:48:39.678113+00:00
[2023-03-28 22:53:44,423] {standard_task_runner.py:52} INFO - Started process 150 to run task
[2023-03-28 22:53:44,425] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'country_dag', 'insert_task', 'manual__2023-03-28T22:48:39.678113+00:00', '--job-id', '298', '--raw', '--subdir', 'DAGS_FOLDER/country_dag.py', '--cfg-path', '/tmp/tmpk3re_4ep', '--error-file', '/tmp/tmpmmrgk26p']
[2023-03-28 22:53:44,426] {standard_task_runner.py:80} INFO - Job 298: Subtask insert_task
[2023-03-28 22:53:44,477] {task_command.py:369} INFO - Running <TaskInstance: country_dag.insert_task manual__2023-03-28T22:48:39.678113+00:00 [running]> on host c0bc2ca16b74
[2023-03-28 22:53:44,549] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Ranga
AIRFLOW_CTX_DAG_ID=country_dag
AIRFLOW_CTX_TASK_ID=insert_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-28T22:48:39.678113+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-28T22:48:39.678113+00:00
[2023-03-28 22:53:45,250] {python.py:173} INFO - Done. Returned value was: None
[2023-03-28 22:53:45,272] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=country_dag, task_id=insert_task, execution_date=20230328T224839, start_date=20230328T225344, end_date=20230328T225345
[2023-03-28 22:53:45,321] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-28 22:53:45,355] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
